xcube/core/store/fs/impl/fs.py

```diff
@@ -136,3 +136,21 @@ def get_storage_options_schema(cls) -> JsonObjectSchema:
             ),
             additional_properties=True,
         )
+
+class AzureFsAccessor(FsAccessor):
+    @classmethod
+    def get_protocol(cls) -> str:
+        return 'abfs'
+
+    @classmethod
+    def get_storage_options_schema(cls) -> JsonObjectSchema:
+
+        return JsonObjectSchema(
+            properties=dict(
+                anon=JsonBooleanSchema(
+                    title='Whether to anonymously connect to Azure Blob Storage.'
+                ),
+                **COMMON_STORAGE_OPTIONS_SCHEMA_PROPERTIES,
```

Comment:
Please specify the Azure-specific store parameters here:

* `account_name`
* `account_key`
* `connection_string`